libp2p/Session.cpp

```diff
@@ -206,6 +206,8 @@ bool Session::checkPacket(bytesConstRef _msg)
 
 void Session::send(bytes&& _msg)
 {
+    if(m_dropped)
```

Comment:
(Nit) Formatting...can you please format your changes using `git clang-format` as described in item 3 here: https://github.com/ethereum/aleth/blob/master/CONTRIBUTING.md